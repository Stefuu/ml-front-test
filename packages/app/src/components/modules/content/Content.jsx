import styled from 'styled-components'
import { theme } from 'styled-tools'
import Box from '../../elements/Box'

const Content = styled(Box)`
  min-height: 100vh;
  background-color: ${props => props.color || theme('colors.b5')};
`

export default Content
