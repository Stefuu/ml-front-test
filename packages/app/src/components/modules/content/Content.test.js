import React from 'react'
import { render } from 'react-testing-library'
import Content from './Content'

describe('Content', () => {
  it('should render [snapshot]', () => {
    const { container } = render(<Content />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
