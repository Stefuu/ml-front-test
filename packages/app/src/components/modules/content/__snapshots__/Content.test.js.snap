// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content should render [snapshot] 1`] = `
<div
  class="sc-bdVaJa sc-bwzfXH kzHfNy"
/>
`;
